src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1931,6 +1959,8 @@ bool AdaptDispatch::ScreenAlignmentPattern()
         auto fillPosition = COORD{ 0, csbiex.srWindow.Top };
         const auto fillLength = (csbiex.srWindow.Bottom - csbiex.srWindow.Top) * csbiex.dwSize.X;
         success = _pConApi->PrivateFillRegion(fillPosition, fillLength, L'E', false);
+        // Reset the line rendition for all of these rows.
+        success = success && _pConApi->PrivateResetLineRenditionRange(csbiex.srWindow.Top, csbiex.srWindow.Bottom);
```

Comment:
Huh, we fill *after* we reset the rendition? I don't totally understand why, but I trust your homework here.

Reply:
erm we fill *before*.

Reply:
I don't think it matters. The fill operation doesn't care about line renditions, so I suspect it would work in either order. I do know that this sequence works correctly as is - it's one of my manual test cases.

Reply:
I've just double checked this and the order doesn't make any difference. In both case the lines are reset to single width and the screen is completely filled with Es.